File: aula-07/vetor/java/src/test/java/com/github/jc00973/ufg/cs/aula07/QuantidadeElementosIguaisTest.java
package com.github.jc00973.ufg.cs.aula07;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertThrows;

public class QuantidadeElementosIguaisTest {

    int[] vetorTeste = {0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0};

    @Test
    void testQuantidadeElementosIguais() {
        assertEquals(4, AlgoritmosVetor.quantidadeElementosIguais(vetorTeste, 1));
    }

}

File: aula-07/vetor/js/test/quantidadeElementosIguais.test.js
const funcoes = require("../AlgoritmosVetor");

test("teste 1", () => {
    expect(funcoes.quantidadeElementosIguais([0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0], 1)).toBe(4);
});

